noodles-editor/src/noodles/components/categories.ts

```diff
@@ -1,118 +1,124 @@
-    'LineLayerOp',
-    'MVTLayerOp',
-    'PathLayerOp',
-    'PointCloudLayerOp',
-    'PolygonLayerOp',
-    'QuadkeyLayerOp',
-    'RasterTileLayerOp',
-    'S2LayerOp',
-    'ScatterplotLayerOp',
-    'ScenegraphLayerOp',
-    'ScreenGridLayerOp',
-    'SimpleMeshLayerOp',
-    'SolidPolygonLayerOp',
-    'TerrainLayerOp',
-    'TextLayerOp',
-    'Tile3DLayerOp',
-    'TileLayerOp',
-    'TripsLayerOp',
+    'A5Layer',
+    'ArcLayer',
+    'BitmapLayer',
+    'ColumnLayer',
+    'ContourLayer',
+    'GeoJsonLayer',
+    'GeohashLayer',
+    'GreatCircleLayer',
+    'GridCellLayer',
+    'GridLayer',
+    'H3ClusterLayer',
+    'H3HexagonLayer',
+    'HeatmapLayer',
+    'HexagonLayer',
+    'IconLayer',
+    'LineLayer',
+    'MVTLayer',
+    'PathLayer',
+    'PointCloudLayer',
+    'PolygonLayer',
+    'QuadkeyLayer',
+    'RasterTileLayer',
+    'S2Layer',
+    'ScatterplotLayer',
+    'ScenegraphLayer',
+    'ScreenGridLayer',
+    'SimpleMeshLayer',
+    'SolidPolygonLayer',
+    'TerrainLayer',
+    'TextLayer',
+    'Tile3DLayer',
+    'TileLayer',
+    'TripsLayer',
   ],
   extension: [
-    'BrightnessContrastExtensionOp',
-    'BrushingExtensionOp',
-    'ClipExtensionOp',
-    'CollisionFilterExtensionOp',
-    'DataFilterExtensionOp',
-    'FillStyleExtensionOp',
-    'HueSaturationExtensionOp',
-    'Mask3DExtensionOp',
-    'MaskExtensionOp',
-    'PathStyleExtensionOp',
-    'TerrainExtensionOp',
-    'VibranceExtensionOp',
+    'BrightnessContrastExtension',
+    'BrushingExtension',
+    'ClipExtension',
+    'CollisionFilterExtension',
+    'DataFilterExtension',
+    'FillStyleExtension',
+    'HueSaturationExtension',
+    'Mask3DExtension',
+    'MaskExtension',
+    'PathStyleExtension',
+    'TerrainExtension',
+    'VibranceExtension',
   ],
-  number: ['NumberOp', 'MapRangeOp', 'ExtentOp', 'MathOp', 'BezierCurveOp', 'TimeOp'],
-  string: ['StringOp'],
+  number: ['Number', 'MapRange', 'Extent', 'Math', 'BezierCurve', 'Time'],
+  string: ['String'],
   utility: [
-    'BooleanOp',
-    'ConsoleOp',
-    'LayerPropsOp',
-    'MouseOp',
-    'ProjectOp',
-    'UnprojectOp',
-    'MapStyleOp',
+    'Boolean',
+    'Console',
+    'LayerProps',
+    'Mouse',
+    'Project',
+    'Unproject',
+    'MapStyle',
   ],
-  vector: ['CombineXYOp', 'CombineXYZOp', 'SplitXYOp', 'SplitXYZOp'],
+  vector: ['CombineXY', 'CombineXYZ', 'SplitXY', 'SplitXYZ'],
   view: [
-    'FirstPersonViewOp',
-    'GlobeViewOp',
-    'MaplibreBasemapOp',
-    'MapViewOp',
-    'MapViewStateOp',
-    'OrbitViewOp',
-    'SplitMapViewStateOp',
+    'FirstPersonView',
+    'GlobeView',
+    'MaplibreBasemap',
+    'MapView',
+    'MapViewState',
+    'OrbitView',
+    'SplitMapViewState',
   ],
-  widget: ['FpsWidgetOp'],
+  widget: ['FpsWidget'],
 } as const
+
+// TODO: Remove this function when we fully migrate to operator displayNames
+export function nodeTypeToDisplayName(nodeType: string): string {
+  return nodeType.replace(/Op$/, '')
+}
```

Comment:
Follow up: #181 switched categories to use `displayName`. Do we also want to migrate ReactFlow and serialized project files to use display names instead of "Op" suffixed types?

Context is `opTypes` keys stringify at build-time to their class name

https://github.com/joby-aviation/noodles.gl/blob/86807a5b813bb483edde5463f8ff99804c22b7ef/noodles-editor/src/noodles/operators.ts#L5303-L5309

And it was assumed `op.constructor.name` would be equivalent in production, but those values get minified. `displayName` already existed and by convention is a unique name we could use to identify an op.